src/Microsoft.VisualStudio.ProjectSystem.Managed.TestServices/AssertEx.cs

```diff
@@ -26,5 +24,30 @@ public static void CollectionLength(IEnumerable collection, int expectedCount)
         {
             CollectionLength(collection.Cast<object>(), expectedCount);
         }
+
+        public static void SequenceEqual<T>(IEnumerable<T> expected, IEnumerable<T> actual)
```

Comment:
Context?

Reply:
I rewrote part of `VisualBasicNamespaceImportsList` to work better with nullability analysis, and that broke `VisualBasicNamespaceImportsListTests.UpdateNamespaceImportListTest`, so I rewrote that test which benefited from this assertion method.